locales/en.json

```
@@ -1,7 +1,7 @@
 {
     "Title": "Contoso Air",
 	"Home": {
-		"HeroMessage": "Where do you<br>want to go?"
+		"HeroMessage": "Where do you really<br>want to go?"
```

```suggestion
		"HeroMessage": "Where do you really<p>want to go?"
```

I think ideally this would be `<p>` vs `<br>`.

:octocat: :heart:

---

I love it, @bitoiu .